src/sinks/datadog/traces/stats.rs

```diff
@@ -0,0 +1,502 @@
+use std::collections::BTreeMap;
```

Comment:
There is a lot going on in this file, but very little in the way of comments explaining what it is doing.

I'd really like to see at least most of the methods here with method level documentation, it's not always apparent from the type definitions or the method names what is expected of each function.